ScattReportTest.java

```diff
@@ -0,0 +1,61 @@
+import org.junit.Test;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import static org.junit.Assert.assertEquals;
+import org.junit.Before;
+import org.junit.After;
+
+/**
+ * ScattTest.java
```

Comment:
This will need to be changed to ReportTest.java (as will the filename to match our naming conventions for other classes).